pytorch_vision_deeplabv3.md

```diff
@@ -2,8 +2,8 @@
 layout: hub_detail
 background-class: hub-background
 body-class: hub
-title: Deeplabv3-ResNet101
-summary: DeepLabV3 model with a ResNet-101 backbone
+title: Deeplabv3
```

Comment:
You did well changing the title here, however I would suggest not to change the file name. I agree with you that it somehow conveys the wrong meaning, but changing the file name will break the existing URL (strangely enough they seem to still exist, but it's even worse, as those URLs will never be properly updated).

That the URL doesn't fully reflect the page content isn't that big of an issue IMO, at least it's not as bad as breaking existing URLs.

I will update our runbook / review guidelines to indicate that we should try to have filenames that are as generic as possible, to avoid such issues in the future

Reply:
Good catch. How about keeping the old url with a link to the new location? Is this possible given our setup?

Reply:
I think we would need to keep the existing `pytorch_vision_deeplabv3_resnet101.md ` file, which is not ideal - it would also show-up needlessly in the hub search results, etc.

One option would be to have an HTTP redirection in place, perhaps with https://github.com/jekyll/jekyll-redirect-from

But I would keep that for further improvements and keep the same filenames for now :)

Reply:
Great then, I have renamed the files. I think it works fine.